rules/terraform_required_version.go

```diff
@@ -78,13 +80,53 @@ func (r *TerraformRequiredVersionRule) Check(runner tflint.Runner) error {
 		exists = exists || ok
 	}
 
-	if !exists {
-		return runner.EmitIssue(
-			r,
-			`terraform "required_version" attribute is required`,
-			hcl.Range{},
-		)
+	if exists {
+		return nil
+	}
+
+	if len(body.Blocks) > 0 {
+		return r.emitIssue(body.Blocks[0].DefRange, runner)
 	}
 
-	return nil
+	// If there are no "terraform" blocks, create a hcl.Range from the files
+	var file string
+	for k := range files {
+		file = k
+		break
+	}
+
+	// If there is only one file, use that
+	if len(files) == 1 {
+		return r.emitIssue(hcl.Range{
+			Filename: file,
+			Start:    hcl.InitialPos,
```

Comment:
We may need to specify the `End` as well.

Reply:
Should that also be `hcl.InitialPos`?

Reply:
Yes, a range where the `End` is less than the `Start` may not be valid.

Reply:
Thanks, done!